### ISSUE TITLE
Inconsistent rendering?

### ISSUE THREAD

**Issue by muralito** [organicmaps/organicmaps#6175]
**Describe the issue**
OM is rendering two pedestrian areas different

**Steps to reproduce**
The area is here https://www.openstreetmap.org/#map=19/-34.91029/-56.20592

**Expected behavior**
Pedestrian areas have the same rendering (similar to the openstreetmap.org map)

**Screenshots**
OM screenshot 
![Screenshot_20231001-223800_Trebuchet](https://github.com/organicmaps/organicmaps/assets/7611868/0c5d0495-1e4d-482e-b2b8-f69465ddd7bd)

Screenshot of the OSM standard style
![image](https://github.com/organicmaps/organicmaps/assets/7611868/57a25ace-23ab-4da6-8ed4-657b7db23e79)

**System information:**
Organic Maps version 2023.99.22-27-fdroid with the maps already updated (20/09/2023)
The same rendering is reproducible in LineageOS 18.1 (android 11) and Android 12.

**Additional context**
Maybe is related to the leisure=park rendering on top of the pedestrian areas.

**Comment by strump** [comment]
You are right. Polygon https://www.openstreetmap.org/relation/16099434 with `leisure=park` is rendered on top of pedestrian zone.

**Comment by pastk** [comment]
> 
> **Expected behavior** Pedestrian areas have the same rendering (similar to the openstreetmap.org map)
> 
> 
> **Additional context** Maybe is related to the leisure=park rendering on top of the pedestrian areas.

Yeap this is exactly the reason and this is how its supposed to be IMO (leisure=park has a smaller bbox than a pedestrian area hence is rendered on top).

There are caveats in this case though:
- the actual area of the pedestrian zone is likely smaller than of the park, 'cause it has some "holes"
- the park is mapped as a relation (maybe that's a reason osm.org doesn't render it?)

Maybe we should ignore area tags set in relations? I doubt it, it's likely to break other cases.

Overall this case reminds of river vs surrounding forest (when river polygons are mapped directly over forest ones and river having a bigger bbox because of its lenght/curves and hence forest polygons rendered over the river). But I don't think making pedestrian zones rendered over parks (or any other green areas) is a viable solution because there might be patches of green mapped over a big pedestrian zone.

Any ideas?

**Comment by pastk** [comment]
@vng WDYT?

**Comment by pastk** [comment]
![image](https://github.com/organicmaps/organicmaps/assets/18434508/be8c7bed-413d-4539-b9eb-9ae79dfb0c74)

**Comment by vng** [comment]
Fancy mapping IMO. 2 options:
- In fact, there are grass and pedestrian area, so leisure=park can be removed
- If one part of the pedestrian area is inside the park and the other one is not, split this pedestrian area in 2 separate areas and include one (as inner) in the park relation.

**Comment by muralito** [comment]
> Fancy mapping IMO. 2 options:
> 
Maybe, but more "detailed" than "fancy" IMO.

>     * In fact, there are grass and pedestrian area, so leisure=park can be removed
> 
Not sure. The park feature exists by itself, has a name, etc.

>     * If one part of the pedestrian area is inside the park and the other one is not, split this pedestrian area in 2 separate areas and include one (as inner) in the park relation.

Maybe it is a tagging mistake, not sure yet.
There are several type of items there, some phisical, like the pedestrian area, the grass green area, etc, and they are inside "the park". The park includes both pedestrians and grass areas.

Maybe the rendering of the park should be a line in its border and the name, but not the interior of the polygon. Or maybe not 100% transparent, but a value that allows to see what it's rendered below it.

**Comment by muralito** [comment]
> But I don't think making pedestrian zones rendered over parks (or any other green areas) is a viable solution because there might be patches of green mapped over a big pedestrian zone.

There are several parks wich are covered by grass and are not green areas. Maybe the leisure=park should keep the deafult of green rendering, but also take in accounts other tags in itself, or in elements contained in their polygons.
